enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk_setup/components/GetCodeStep.tsx

```diff
@@ -118,7 +118,10 @@ export const GetCodeStep = () => {
                 leftSection={<Icon name="copy" size={16} />}
                 onClick={() => {
                   copy();
-                  trackEmbedWizardCodeCopied();
+
+                  trackEmbedWizardCodeCopied(
```

Comment:
Could/should we track the ctrl-c event? Not every user will trust the copy button and use it btw. [Ref](https://metaboat.slack.com/archives/C063Q3F1HPF/p1757498765171239?thread_ts=1754802429.273019&cid=C063Q3F1HPF).

Reply:
Fixed in https://github.com/metabase/metabase/pull/63973/commits/aa795c7548ffc9b8eae0b962fd943d65240e1910

Reply:
Fixed in https://github.com/metabase/metabase/pull/63973/commits/1c36d919f0d929bb6d17c20f5d099fca35abd1b0